### PR TITLE
Chemical files docs

### PR DESCRIPTION
**Pull request by fiskus**

### CODE REVIEW

File: docs/Catalog/Preview.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ Quilt can display any plaintext file format, including the following.
 * Biological file formats (.bed, .cef, .gff, .fasta, .fastq, .sam, .pdbqt, .vcf, etc.)
 * Text files (.csv, .md, .readme, .tsv, .txt, etc.)
 
+## Chemical files
+We use NGL library to render chemical files. It can't render v3000 Mol file format, so we convert it to v2000 on-the-fly.
```
Comment (akarve) [inline docs/Catalog/Preview.md:17]: The Quilt catalog uses the [NGL Viewer library](https://github.com/nglviewer/ngl) to render structures.
By default, v3000 Molfiles are converted to v2000 by the JavaScript client for rendering.

File: docs/Catalog/Preview.md [hunk 2]
```diff
@@ -13,6 +13,14 @@ Quilt can display any plaintext file format, including the following.
...
+
+It's a work in progress, preview for .mol files is better developed
+
+* Mol files (.mol, .mol2, .sdf)
+* Other chemical files (.cif, .ent, .pdb)
```
Comment (akarve) [inline docs/Catalog/Preview.md:22]:
```suggestion
## Chemical structures
The Quilt catalog uses the [NGL Viewer library](https://github.com/nglviewer/ngl) to render structures.
By default, v3000 Molfiles are converted to v2000 by the JavaScript client for rendering.

The following file formats are supported:
* Mol files (.mol, .mol2, .sdf)
* .cif
* .ent
* .pdb
```

**Review by akarve — APPROVED** [review]
Approved pending merge of proposed changes and udpating (e.g. to add mention of JSMol?).

**Comment by fiskus** [comment]
I think JSMol is completely out of scope now, so we shouldn't mention it

**Pull request merged**